CONTRIBUTING.md

```diff
@@ -26,16 +26,16 @@ Config branches are entirely separate from the `main` history in this repository
 
 ### Brand new configuration
 
-If you are creating a brand new configuration, and don't have the config stored in another repository, just checkout a `dev-*` branch from `main` and delete everything except `.github/workflows/pr-1-ci.yml`, `.github/workflows/pr-3-bump-tag.yml` and `.github/workflows/validate-json.yml`, then add your config.
+If you are creating a brand new configuration, and don't have the config stored in another repository, just checkout a `dev-*` branch from `main` and delete everything except `.github/workflows/ci.yml` and `.github/workflows/schedule.yml`, then add your config.
```

Comment:
I thought `schedule.yml` was just on main branch?

Reply:
Whoops, you're right. Fixed in https://github.com/ACCESS-NRI/access-om2-configs/pull/117/commits/04002d09eca489a2112c86e609242dc950d214f8